fern/pages/api-definition/fern-definition/depending-on-other-apis.mdx

```diff
@@ -23,26 +23,44 @@ $ fern register
 
 ## Depending on the registered API
 
-To add a dependency on another API, you simply create a folder in your Fern
-Definition to "house" the dependency. Inside the folder, create a special file
+To add a dependency on another API, you must add a `dependencies.yml` to declare which 
+APIs you wish to depend on. 
+
+```yaml {4}
+fern/
+├─ fern.config.json
+├─ generators.yml
+├─ dependencies.yml
+└─ definition/
+  ├─ api.yml
+  ├─ imdb.yml
+```
+
+Your `dependencies.yml` has a list of all the APIs you wish to depend: 
+
+```yaml dependencies.yml 
+dependencies:
+   "@fern/some-dependency": "0.0.1"
+```
+
+Next, you need create a folder in your Fern Definition to "house" the dependency. Inside the folder, create a special file
```

Comment:
Maybe to `store` the dependency? Maybe `Next, the dependency will need to be stored in a folder within your Fern Definition`?

Reply:
Talked offline -- agreed to take out the quotes for `house`